### DISCUSSION TITLE
https://thanhle.blog/blog/passive-learning-cho-dev

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#21] [Ideas]
# https://thanhle.blog/blog/passive-learning-cho-dev

https://thanhle.blog/blog/passive-learning-cho-dev

**Comment by nguyenhoaikhang37** [comment]
1 chiếc bài viết hay ho, em thì có thói quen hay vào blog anh vào lúc muốn thư giản và còn biết thêm về tech, mong anh ra nhiều bài viết hơn nữa (vd 2 bài viết / 1 tuần ) thay vì 1 ^^

**Reply by thanhlmm** [reply]
Vậy tập thêm đọc từ nhiều nguồn khác nữa em, từ từ là sẽ thấy mình khác bọt ngay.
1 tuần 1 bài cũng là khá khó với anh rồi, lâu lâu bí quá đâu biết viết gì đâu :D, hy vọng sau này viết được nhiều hơn
